src/progress_bar.jl

```diff
@@ -1,6 +1,6 @@
 export ProgressBar, next!
 
-struct ProgressBar{CT,T1<:Integer,T2<:Real,T3,T4<:Real,LT}
+mutable struct ProgressBar{CT,T1<:Integer,T2<:Real,T3,T4<:Real,LT}
```

Comment:
This change should be in another PR.
maybe you need to merge from main branch locally first.

Reply:
I have just done "merge from upstream:main", but now it seems a like a change